src/targets/awsLambdaLayer.ts

```diff
@@ -168,8 +168,12 @@ export class AwsLambdaLayerTarget extends BaseTarget {
 
         await git.add(['.']);
         await git.checkout('master');
+        const runtimeNames = this.config.compatibleRuntimes.map(
+          (runtime: CompatibleRuntime) => runtime.name
+        );
         await git.commit(
-          `craft: AWS Lambda layers published, version ${version}`
+          'craft(aws-lambda): AWS Lambda layers published\n\n' +
+            `v${version} for ${runtimeNames}`
```

Comment:
Seriously, why do you need string concatenation with template strings (backtick strings)? 😁